javaslang/src/main/java/javaslang/collection/Array.java

```diff
@@ -662,23 +662,40 @@ public T get(int index) {
     }
 
     @Override
-    public Array<T> dropUntil(Predicate<? super T> predicate) {
+    public Array<T> dropWhile(Predicate<? super T> predicate) {
         Objects.requireNonNull(predicate, "predicate is null");
-        return dropWhile(predicate.negate());
+        return dropUntil(predicate.negate());
     }
 
     @Override
-    public Array<T> dropWhile(Predicate<? super T> predicate) {
+    public Array<T> dropUntil(Predicate<? super T> predicate) {
```

Comment:
I rearranged the methods to: `drop`, `dropUntil`, `dropWhile`, `dropRight`, `dropRightUntil`, `dropRigtWhile`, it looks more logical to me, what do you think?